src/types.d.ts

```diff
@@ -51,4 +51,5 @@ export interface ISigner {
 export interface IInpageAccountWrapper {
   account: InpageAccount;
   error: Error;
+  statusChangeReason: QRYPTO_ACCOUNT_CHANGE;
```

Comment:
forgot to import the `constant`

Reply:
hmm interesting that the linter/vscode didnt complain and seems to work without it - any idea why?  something special about types.d.ts?